README.md

```diff
@@ -8,6 +8,8 @@ Ensure the you have the following installed:
 - Vagrant
 - Ansible
 - Hostupdater (`vagrant plugin install vagrant-hostsupdater`)
+- Composer
+- Puli
```

Comment:
I was unable to run `composer install` without having puli installed globally. I'm not sure this is ideal. Having as few dependencies on the host will make it easier for those not working on a mac to make this project run. I think the only way around this is to have 2 composer files. One that we leave in the root which just sets up the drupal-vm project and then another which can be run from inside the vm.

This is non-blocking.